src/rpminspect.c

```diff
@@ -267,18 +261,20 @@ static char *get_product_release(string_map_t *products, const favor_release_t f
     free(before_product);
 
     /* trim the leading period */
-    i = 0;
+    if (after_product) {
+        i = 0;
```

Comment:
Preexisting, but `i` was already 0 here, regardless of the value of `after_product`.

Reply:
Thanks